backend/src/main/java/com/gaucho/sos/security/services/JwtService.java

```diff
@@ -0,0 +1,65 @@
+package com.gaucho.sos.security.services;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTCreationException;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.gaucho.sos.domain.User;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
```

Comment:
Daria pra remover linha em branco aqui também. 
```suggestion
import java.time.Instant;
```

Reply:
Alinhado que vamos manter dessa forma.